packages/salesforcedx-vscode-core/src/orgPicker/orgList.ts

```diff
@@ -172,6 +173,11 @@ export class OrgList implements vscode.Disposable {
         vscode.commands.executeCommand('sfdx.force.auth.accessToken');
         return { type: 'CONTINUE', data: {} };
       }
+      case '$(plus) ' +
```

Comment:
minor point (and nothing to do w/your PR), do you mind changing lines 156-159 to
```
return { type: 'CONTINUE', data: {} };
```
(this will match the other cases)